easybuild/easyconfigs/g/git-lfs/git-lfs-1.1.1.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = "Tarball"
+
+name = 'git-lfs'
+version = '1.1.1'
+
+homepage = 'https://git-lfs.github.com/'
+description = """Git Large File Storage (LFS) replaces large files such as audio samples, videos, 
+ datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server 
+ like GitHub.com or GitHub Enterprise."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/github/git-lfs/releases/download/v%(version)s/']
+sources = ['%(name)s-linux-amd64-%(version)s.tar.gz']
+
+sanity_check_paths = {
+    'files': ["git-lfs"],
+    'dirs': [""]
+}
+
+# add the installation dir to PATH
+modextrapaths = {
+      'PATH': '',
+      }
+
+# delete to provide installation script
+postinstallcmds = ["rm -fr %(installdir)s/install.sh"]
```

Comment:
the install.sh script has execution permission so when adding the install_dir to PATH it would be accessible for users and I prefer to avoid it

Also, this is the content of this script which tries to install to /usr/local 

```
#!/usr/bin/env bash
set -eu

prefix="/usr/local"

if [ "${PREFIX:-}" != "" ] ; then
  prefix=${PREFIX:-}
elif [ "${BOXEN_HOME:-}" != "" ] ; then
  prefix=${BOXEN_HOME:-}
fi

mkdir -p $prefix/bin

rm -rf $prefix/bin/git-lfs*
for g in git*; do
  install $g "$prefix/bin/$g"
done

PATH+=:$prefix/bin
git lfs install
```

I was thinking about adding a ```modloadmsg``` line like 'to initialize git-lfs execute git lfs install. This is only needed once' but I decided not to add it. Do you think it's worth?